trailblazer/dto/analyses_response.py

```diff
@@ -18,6 +18,7 @@ class Analysis(BaseModel):
     comment: str | None = None
     completed_at: datetime | None = None
     config_path: str | None = None
+    delivered: bool | None = None
```

Comment:
Maybe we can also just initialise as False, then we can skip none checks? I mean ideally it is either true or false

Reply:
False currently means that the existing entry gets dropped, so I don't think that is an option currently 😞